go_lib/hooks/tls_certificate/internal_tls.go

```diff
@@ -35,19 +35,21 @@ const (
 	// ca expiration = 10 years
 	caExpiryDurationStr = "87600h"
 	// certificate expiration - 10 years
-	certExpiryDuration = 87600 * time.Hour
+	certExpiryDuration = 10 * 365 * 24 * time.Hour
```

Comment:
Honestly. It doesn't make this record more obvious but makes it different with the record which is 2 lines above.
We had "87600h" and 87600 * time.Hour - you can get that they are equal
but in this case you have calculate something to get that they are equal

Reply:
I would write
```
	caExpiryDurationStr = "87600h" // 10 years
	certExpiryDuration = 87600 * time.Hour // 10 years
```

Reply:
Yup, made comments more compact, since the naming is clean enough